treestats_paper.tex

```diff
@@ -239,7 +264,7 @@ \section*{Framework and statistics}
 \begin{align*}
     \sum_p h(p) f(p) = \sum_{i=1}^L \sum_a f(p_i(a)).
 \end{align*}
-(Note that since the ancestral allele is included, this function is invariant to relabeling of alleles.)
+% (Note that since the ancestral allele is included, this function is invariant to relabeling of alleles.)
```

Comment:
we need to emphasize here that the ancestral allele is included, since many people will first read this "sum over a function of (derived) allele frequencies" since that is standard (aaron made this mistake)

Reply:
we talk about this below, but can't emphasize too many times, i think

Reply:
OK, maybe we can work it in some other way? It just seemed off the point here to talk about invariance to relabeling of alleles. I'm not a big fan of parenthetical sentences in the middle of a paragraph, it kills any rhythm, and looks like an afterthought shoe-horned in.